tests/testthat/test-conv.R

```diff
@@ -6,5 +6,7 @@ test_that("encoding conversion works", {
 })
 
 test_that("check encoding argument", {
-  expect_error(str_conv("A", c("ISO-8859-1", "ISO-8859-2")), "single string")
+  expect_snapshot(error = TRUE,
```

Comment:
When there's a single expression to test could you please fit on one line like this? 

```R
expect_snapshot(str_conv("A", c("ISO-8859-1", "ISO-8859-2")), error = TRUE)
```